thinkhazard/static/less/report.less

```diff
@@ -252,6 +252,21 @@ body {
   }
 }
 
+footer {
+  display: table;
+  white-space: nowrap;
+  &:before, &:after {
+    border-top: 5px solid #e6e6e6;
+    content: '';
+    display: table-cell;
+    position: relative;
+    top:0.5em ;
```

Comment:
The problem is that calc doesn't work the way you expect in this case.
I just push a new commit to fix the problem.